tests/integration_test/access_log_test.go

```diff
@@ -61,37 +62,62 @@ func TestAccessLog(t *testing.T) {
 	// https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators
 	accessLogFormat := "\"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\"" +
 		"%RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT%" +
-		"\"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\"" +
-		"\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" " +
-		"%FILTER_STATE(com.google.espv2.filters.http.path_matcher.operation):60% " +
-		"%FILTER_STATE(com.google.espv2.filters.http.service_control.api_key):30%\n"
+		"\"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" " +
+		"%FILTER_STATE(com.google.espv2.filters.http.path_matcher.operation):70% " +
```

Comment:
in prepare removing path_matcher,  can we rename this filter_state key as 
"com.google.espv2.filters.http.service_control.api_method"
before it is used by api gateway. 

We can do it in another pr.

Reply:
ACK.

Reply:
b/162231154